apps/src/music/views/MusicView.jsx

```diff
@@ -726,6 +724,14 @@ class UnconnectedMusicView extends React.Component {
 
 const MusicView = connect(
   state => ({
+    isActive:
```

Comment:
If we do want to go this route, I think this prop should be passed in at the top level by LabContainer directly, or this calculation should get encapsulated in a selector somewhere

Reply:
Agreed.  At least for now, it doesn't seem to be necessary.